report/analysis.py

```diff
@@ -182,20 +183,169 @@ def _batch_bins_and_data(batches, values, function_to_apply):
     return batch_bins, data
 
 
+formats = collections.OrderedDict(
+    [
+        ("High resolution limit", " %7.2f"),
```

Comment:
Humbly ask if this static format is appropriate for very high resolution small molecule data?